ckanext/opendata_theme/processor.py

```diff
@@ -0,0 +1,397 @@
+from abc import abstractmethod
+from collections import defaultdict, OrderedDict
+import wcag_contrast_ratio as contrast
+
+__all__ = ['custom_style_processor']
+
+from ckanext.opendata_theme.color_conrast import get_contrast
+
+
+class AbstractParser:
```

Comment:
must be inherited from ABC class for preventing accidental initialization of the abstract class

Reply:
Yeah, we should change it to import ABC and inherit it.